rosetta-lang/src/main/java/com/regnosys/rosetta/types/RosettaTypeProvider.java

```diff
@@ -84,18 +85,23 @@ public RMetaAnnotatedType getRMetaAnnotatedType(AnnotationPathExpression expr) {
     }
 
     public List<RMetaAttribute> getRMetaAttributesOfSymbol(RosettaSymbol symbol) {
+    	List<RMetaAttribute> acc = new ArrayList<>();
         if (symbol instanceof Attribute a) {
             if (a.isOverride()) {
-                List<RMetaAttribute> acc = new ArrayList<>();
+                
                 acc.addAll(getRMetaAttributesOfSymbol(extensions.getParentAttribute(a)));
-                acc.addAll(getRMetaAttributes(a.getAnnotations()));
-                return acc;
+                acc.addAll(getRMetaAttributes(a.getAnnotations())); 
             }
+            
+        	RosettaType attributeType = a.getTypeCall().getType();
+        	if (attributeType instanceof Data data) {
+        		acc.addAll(getRMetaAttributesOfType(data));
+        	}
         }
         if (symbol instanceof Annotated ann) {
-            return getRMetaAttributes(ann.getAnnotations());
+            acc.addAll(getRMetaAttributes(ann.getAnnotations()));
```

Comment:
One more thing - if I'm correct, then an attribute is also an annotated. So for an attribute, I think these metaattributes will now be added twice? Both on line 93 and 102. Just checking.